### PR TITLE
fix(universal): add using global window if exists

### PR DESCRIPTION
**Pull request by MillerSvt**
Closes #

### CODE REVIEW

File: libs/universal/src/constants/universal-window.ts [hunk 1]
```diff
@@ -58,6 +58,10 @@ const DB_REQUEST: IDBOpenDBRequest = {
 const SELF = ['frames', 'parent', 'self', 'top', 'window'];
 const WINDOW_HANDLER: ProxyHandler<Window> = {
     get: (windowRef, key: string) => {
+        if (typeof window !== 'undefined' && Reflect.has(window, key)) {
```
Comment (waterplea) [inline libs/universal/src/constants/universal-window.ts:61]: Shouldn't it be `globalThis`? I believe it's already available in all our supported browsers

**Comment by splincode** [comment]
@MillerSvt any progress?

**Comment by MillerSvt** [comment]
> @MillerSvt any progress?

I'm not sure that this is necessary. The original problem was solved.

**Comment by waterplea** [comment]
> > @MillerSvt any progress?
> 
> I'm not sure that this is necessary. The original problem was solved.

Right, let's not introduce changes then, if they are not needed at the moment.

**Pull request closed without merging**